components-core/src/main/java/com/adyen/checkout/components/model/CountryInfo.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 18/12/2020.
+ */
+
+package com.adyen.checkout.components.model
+
+/**
+ * Class that holds data about a given country
+ */
+data class CountryInfo(
```

Comment:
The `model` package, so far, is meant for classes that extend `ModelObject` and mirror the requests and responses from the Checkout API.

Do you think it would be better to have `CountryInfo` and `CountryUtils` in a single file? They seem pretty related to me.

Reply:
👍